perch/views.py

```diff
@@ -230,10 +230,7 @@ def reference_links(doc):
     if doc.get('type') == 'organisation':
         for asset_id_type, link in doc.get('reference_links', {}).items():
             value = {
-                'organisation_id': doc['_id'],
-                'links': {
-                    asset_id_type: link
-                }
+                'organisation_id': doc['_id']
```

Comment:
Please remove this change, it is unrelated to the other changes and it will break the accounts service API